.github/workflows/pr-test-rust.yml

```
@@ -14,27 +14,45 @@ on:
       - "docs/**"
       - "mkdocs.yml"
       - "*.md"
+  # Allow maintainers to trigger CI for fork PRs by adding the 'ci-approved' label.
+  # pull_request_target runs in the base branch context, bypassing the fork approval gate.
+  pull_request_target:
+    branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)